### PR TITLE
Added plain print functionality and getopts handling

### PR DESCRIPTION
**Pull request by invention64**
Fixes issue #2, also adds a couple more insults.

### CODE REVIEW

File: libwhatami/data/quotes [hunk 1]
```diff
@@ -39,3 +39,11 @@ If I could be precise in my description of you, I say you're a blackhole of inte
...
 Seriously, I think you might need to get this checked out - why do you keep using me?
 DISCLAIMER: I am not responsible for your dumb. 
+You are a nincompoop
+You're a little pissbaby
+You think you are so cool? I think otherwise.
+Did it hurt when you fell from heaven? Because your face is pretty messed up.
+I bet you can't even read
+Go back to typing school you ignoramus!
+You make me wish I was never born
+You my friend, are the biggest asshole I have ever met
```
Comment (kz6fittycent) [inline libwhatami/data/quotes]: Won't be including any language that might be considered inappropriate for children.

File: libwhatami/__init__.py [hunk 2]
```diff
@@ -32,3 +32,7 @@ def main():
 
     with open(ascii_path) as ascii_file:
         print(ascii_file.read())
+
+def plainPrint():
```
Comment (kz6fittycent) [inline libwhatami/__init__.py:36]: Cool! This is much simpler than what I was thinking.

**Review by kz6fittycent — CHANGES_REQUESTED** [review]
I like it. Just need to make sure that language is kept clean enough for kids. Won't budge on that one.

```
libwhatami/data/quotes - 
#43
#49

Remove those and I'll merge. 

Thanks for the PR - this is much better implementation than I was thinking.

File: libwhatami/data/quotes (hunk 1 above)
Comment (invention64) [reply libwhatami/data/quotes]: Yeah I figured based on what you already had

File: libwhatami/__init__.py (hunk 2 above)
Comment (invention64) [reply libwhatami/__init__.py:36]: Thanks!

**Pull request merged**